c2corg_ui/static/js/alertservice.js

```diff
@@ -120,11 +120,16 @@ app.Alerts.prototype.formatErrorMsg_ = function(response) {
   if (len > 1) {
     var msg = '<ul>';
     for (var i = 0; i < len; i++) {
-      msg += '<li>' + this.filterStr_(errors[i]['description']) + '</li>';
+      if (errors[i]['description'] && errors[i]['name']) {
+        msg += '<li><span translate>' + this.filterStr_(errors[i]['description']) +
+                        '</span><span translate> ' + this.filterStr_(errors[i]['name']) +
+                        '<span>' +
+                      '</li>';
+      }
     }
     return msg + '</ul>';
   }
-  return this.filterStr_(errors[0]['description']);
+  return this.filterStr_(errors[0]['description'] + ' at ' + errors[0]['name']);
```

Comment:
This is still not possible to translate using this syntax. And do we need the "at"? It makes things even more complicated. What about something like:
```
return this.filterStr_(errors[0]['description']) + ' (' +  this.filterStr_(errors[0]['name']) + ')';
```
or
```
return '{description} ({error})'
  .replace('{description}', this.filterStr_(errors[0]['description']))
  .replace('{error}', this.filterStr_(errors[0]['error']));
```